### ISSUE TITLE
setupTestFrameworkScriptFile breaks test reporting

### ISSUE THREAD

**Issue by andrew-w-ross** [wallabyjs/public#763]
### Issue description or question

If I add a `setupTestFrameworkScriptFile` option wallaby stops reporting how many tests passed or failed but still highlights what portions of the test ran. If I remove the `setupTestFrameworkScriptFile` option  it reports correctly.

### Wallaby.js configuration file

```javascript

module.exports = function (wallaby) {

    return {
        files: [
            "src/**/*.ts",       
            "!src/**/*.espec.ts",                        
            { pattern: 'spec/**/*', instrument: false, load: true }
        ],
        tests: [
            'src/**/*.espec.ts'
        ],
        compilers: {
            '**/*.ts': wallaby.compilers.typeScript()
        },
        testFramework: 'jest',
        env: {
            type: 'node',
            runner: 'node'
        },
        workers: {
            recycle: false
        },
        setup: function (wallaby) {
            var jestConfig = require("./spec/jest-config.json");
            wallaby.testFramework.configure(jestConfig);
        }
    };
};
```

### jest-config configuration file

```json
{
	"automock": false,
	"setupTestFrameworkScriptFile":"./spec/jest-setup.js",
	"scriptPreprocessor": "./spec/jest-preprocessor.js",
	"testRegex": ".*\\.espec\\.ts$",
	"testEnvironment":"node",
	"moduleFileExtensions": [
		"js",
		"json",
		"ts"		
	],
	"testPathDirs": [
		"src/"
	]	
}
```

### jest-setup.js

```javascript
jasmine.DEFAULT_TIMEOUT_INTERVAL = 40000;
```

### Code editor or IDE name and version

Visual Studio Code v1.4.0

### OS name and version

OSX

**Comment by ArtemGovorov** [comment]
Thanks for the report, the issue is fixed in core 1.0.283.

**Comment by andrew-w-ross** [comment]
@ArtemGovorov Thanks it works, also on a side note the update for the output is excellent.
